fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckFeature.kt

```diff
@@ -13,22 +13,27 @@ import mozilla.components.browser.state.selector.selectedTab
 import mozilla.components.browser.state.store.BrowserStore
 import mozilla.components.lib.state.ext.flowScoped
 import mozilla.components.support.base.feature.LifecycleAwareFeature
+import org.mozilla.fenix.components.AppStore
 
 /**
  * Feature implementation that provides review quality check information for supported product
  * pages.
  *
+ * @property appStore Reference to the application's [AppStore].
  * @property browserStore Reference to the application's [BrowserStore].
  * @property shoppingExperienceFeature Reference to the [ShoppingExperienceFeature].
  * @property onAvailabilityChange Invoked when availability of this feature changes based on feature
  * flag and when the loaded page is a supported product page.
  */
 class ReviewQualityCheckFeature(
+    private val appStore: AppStore,
     private val browserStore: BrowserStore,
     private val shoppingExperienceFeature: ShoppingExperienceFeature,
     private val onAvailabilityChange: (isAvailable: Boolean) -> Unit,
+    private val onBottomSheetCollapsed: () -> Unit,
 ) : LifecycleAwareFeature {
     private var scope: CoroutineScope? = null
+    private var expandedScope: CoroutineScope? = null
```

Comment:
❓ Could we somehow use one same scope for collecting both flows?

Reply:
I don't believe so in this case, since the scopes are being started from two different stores.